queries/luau/indents.scm

```diff
@@ -0,0 +1,5 @@
+; inherits: lua
+
+[
+  (object_type)
```

Comment:
Is there any other case where types should be indented? jfc

Reply:
maybe array types but I severely doubt anyone putting that on a newline

{
  [number]: T
}

Reply:
Hmm, okay then